libs/sdk-common/src/breez_server.rs

```diff
@@ -29,18 +31,26 @@ pub struct BreezServer {
 impl BreezServer {
     pub fn new(server_url: String, api_key: Option<String>) -> Result<Self> {
         Ok(Self {
-            grpc_channel: Mutex::new(Endpoint::from_shared(server_url.clone())?.connect_lazy()),
+            grpc_channel: Mutex::new(Self::create_endpoint(&server_url)?.connect_lazy()),
             api_key,
             server_url,
         })
     }
 
     pub async fn reconnect(&self) -> Result<()> {
-        *self.grpc_channel.lock().await =
-            Endpoint::from_shared(self.server_url.clone())?.connect_lazy();
+        *self.grpc_channel.lock().await = Self::create_endpoint(&self.server_url)?.connect_lazy();
         Ok(())
     }
 
+    fn create_endpoint(server_url: &str) -> Result<Endpoint> {
+        Ok(Endpoint::from_shared(server_url.to_string())?
+            .http2_keep_alive_interval(Duration::new(5, 0))
+            .tcp_keepalive(Some(Duration::from_secs(5)))
```

Comment:
The default keepalive interval is 20 seconds. Is it necessary to put this to 5 seconds?

Reply:
Yes I think it is better and more tailor made for mobile

Reply:
Ok sounds good.